app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/initialgainorloss/summary.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import cats.instances.long._
+@import cats.syntax.eq._
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.initialgainorloss.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.finance.MoneyUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.InitialGainOrLossAnswers.CompleteInitialGainOrLossAnswers
+
+
+@this(
+        cyaSection: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_section,
+        cyaRow: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_row,
+        cyaChange: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_change,
+)
+
+@(answers: CompleteInitialGainOrLossAnswers)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
+
+@initialGainOrLossLabelWithAmount = @{
+  Some(answers.initialGainOrLoss).map { amount =>
+    if(amount.value > 0)
+      "initialGainOrLoss.gain.label" -> Some("taxableGain.label" -> MoneyUtils.formatAmountOfMoneyWithPoundSign(amount.inPounds()))
+    else if(amount.value < 0)
+      "initialGainOrLoss.loss.label" -> Some("netLoss.label" -> MoneyUtils.formatAmountOfMoneyWithPoundSign(amount.inPounds() * -1))
+    else
+      "initialGainOrLoss.noLossOrGain.label" -> None
+  }
+
+}
+
+@cyaSection() {
+
+  @initialGainOrLossLabelWithAmount.map( l =>
+    cyaRow(
+      messages("initialGainOrLoss.title"),
+      Html(messages(l._1)),
+      Some(cyaChange("", routes.InitialGainOrLossController.changeInitialGainOrLoss().url)),
```

Comment:
this empty string should be populated by something - it's what will get read of by screen readers. It will say `change blah blah blah` where `blah blah blah` is what the argument maps to. We need to do a wider exercise to get the right content for that, but for now it's better if it's in the messages file so we can see it there

Reply:
Done.